src/pages/index.js

```diff
@@ -74,6 +75,7 @@ function Index() {
           />
         </div>
       </div>
+      <DappsGallery />
```

Comment:
This needs to be in a container this is why it goes too wide, so either add a new div with container class or add it inside the previous div